app/src/main/res/xml/appfilter.xml

```diff
@@ -3083,9 +3092,6 @@
     <!-- Voice -->
     <item component="ComponentInfo{de.ph1b.audiobook/de.ph1b.audiobook.features.MainActivity}" drawable="voice" />
 
-    <!-- Voice -->
-    <item component="ComponentInfo{de.ph1b.audiobook/de.phib.audiobook.features.MainActivity}" drawable="voice" />
```

Comment:
Why was this one removed?

Reply:
Back when I made the change I compared both existing paths with the one voice uses on my phone. However you are right there may be a different version around. I have now added it back.

Reply:
Ah, I didn't notice the other entry. You're probably right! We can leave them both though.